### PR TITLE
Update Portuguese translation

### PR DESCRIPTION
**Pull request by hugok79**
Minor spelling corrections

### CODE REVIEW

File: po/pt.po [hunk 1]
```diff
@@ -829,7 +829,7 @@ msgstr "Selecionar"
 #: src/modules/quickaccess/e_mod_quickaccess.c:381
 #: src/modules/shot/e_mod_main.c:316 src/modules/shot/e_mod_main.c:469
 #: src/modules/shot/e_mod_main.c:732 src/modules/shot/e_mod_main.c:758
-#: src/modules/shot/e_mod_main.c:998 src/modules/xkbswitch/e_mod_config.c:529
+#: src/modules/shot/e_mod_main.c:998 src/modules/xkbswitch/e_mod_config.c:559
 msgid "Cancel"
 msgstr "Cancelar"
```
Comment (hugok79) [inline po/pt.po:835]: this should happen? Just updated from the .pot file!

**Comment by hugok79** [comment]
Updated the string from the .pot file.
In the meantime I noticed that the file changed something other than the spelling corrections!!

**Review by hugok79 — COMMENTED** [review]
@thewaiter Why did the strings change the number? Sorry, I'm not a programmer :)

File: po/pt.po (hunk 1 above)
Comment (thewaiter) [reply po/pt.po:835]: hmm, not sure. I need to compile, switch to portugese and test if OK

Comment (hugok79) [reply po/pt.po:835]: Ok.

**Comment by thewaiter** [comment]
Looks fine here
https://i.imgur.com/rWpJGnN.jpg
https://i.imgur.com/ycVbRRK.jpg

**Comment by hugok79** [comment]
So the pt.po filet stays that way, right?
Normally and practically in every project I translate, the developer/upstream always updates the **.pot** file when he adds new strings and as such all **.po** files have to be updated with the new strings added.

**Comment by thewaiter** [comment]
OK, my learning skills in progress. Next time I will update pot file also with all po file. Thanx for reminder

**Pull request merged**